android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/CustomButton.kt

```diff
@@ -0,0 +1,45 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.Black
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.tooling.preview.Preview
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.shapes
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.typography
+
+@Composable
+fun CustomButton(
```

Comment:
It's better to avoid words with unclear meanings. I prefer to use `PrimaryButton` instead. Furthermore, if we have to customize another button (e.g., `SecondaryButton`), using `Custom` will not be suitable for expanding.

We have [this convention](https://nimblehq.co/compass/development/code-conventions/swift/#naming-classes-should-be-straightforward) on Swift page, but I think it's ok to mention here :D

Reply:
@suho Good point, fixed in https://github.com/luongvo/kmm-survey/pull/48/commits/2bf5858983ff990d311a04c1e83df985919b6207